backend/src/openarchiefbeheer/destruction/tests/e2e/features/test_feature_list_select_all.py

```diff
@@ -0,0 +1,76 @@
+# fmt: off
+from django.test import tag
+
+from asgiref.sync import sync_to_async
+
+from openarchiefbeheer.destruction.models import DestructionList
+from openarchiefbeheer.utils.tests.e2e import browser_page
+from openarchiefbeheer.utils.tests.gherkin import GherkinLikeTestCase
+
+
+@sync_to_async()
+def get_list(name):
+    return DestructionList.objects.get(name=name)
+
+
+@tag("e2e")
+class FeatureListCreateSelectAllTests(GherkinLikeTestCase):
+    async def test_scenario_record_manager_creates_list_with_select_all(self):
+        async with browser_page() as page:
+            await self.given.record_manager_exists()
+            await self.given.reviewer_exists(username="Beoordelaar 1")
+            await self.given.reviewer_exists(username="Beoordelaar 2")
+            await self.given.zaken_are_indexed(200)
+
+            await self.when.record_manager_logs_in(page)
+            await self.then.path_should_be(page, "/destruction-lists")
+
+            await self.when.user_clicks_button(page, "Vernietigingslijst opstellen")
+            await self.then.path_should_be(page, "/destruction-lists/create")
+
+            await self.when.user_clicks_checkbox(page, "(de)selecteer 200 pagina's", index=0)
```

Comment:
Note: this should be changed when https://github.com/maykinmedia/admin-ui/issues/126 is fixed